makefiles/S3.toml

```diff
@@ -33,15 +33,22 @@ args = [
 
 [tasks.publish-fluvio]
 dependencies = [
-    "build-release",
+    "build-prod",
     "install-fluvio-package",
     "publish-fluvio-env",
 ]
-script = "${HOME}/.fluvio/bin/fluvio package --test publish --version=${FLUVIO_VERSION} ${FORCE_RELEASE} ./target/${TARGET}/release/fluvio"
+env = { FLUVIO_VERSION = { script = ["cat VERSION"] } }
```

Comment:
Why changing this?
Injecting env make this less reusable

Reply:
Publishing Fluvio always requires knowing the contents of the `VERSION` file and this is an elegant way to do it. This is the "default" `publish-fluvio` task, which uses the VERSION directly, there is also another task, `publish-fluvio-latest` which simply overrides FLUVIO_VERSION to include both the VERSION + the git commit.

See https://github.com/infinyon/fluvio/pull/892/files/038e3333c72f9fee0c46ebbd9cf02e91731ea6c0#diff-2dcdc917058b2578ec902f5e1691c7c4c6ce3da1e59036f159ee6230548c4adfR57

Reply:
fair enough.